src/autocomplete/databases/postgresql/tests/comment/comment-constraint.test.ts

```diff
@@ -1,14 +1,16 @@
 import {parsePostgreSqlQueryWithCursor} from '../../../../shared/parse-query-with-cursor';
 import {parsePostgreSqlQueryWithoutCursor} from '../../../../autocomplete';
 
-test('should suggest properly after COMMENT ON CONSTRAINT', () => {
+// TODO Get context of table in COMMENT statement
+test.skip('should suggest properly after COMMENT ON CONSTRAINT', () => {
     const autocompleteResult = parsePostgreSqlQueryWithCursor('COMMENT ON CONSTRAINT |');
 
     expect(autocompleteResult.suggestKeywords).toEqual([]);
     expect(autocompleteResult.suggestConstraints).toEqual(true);
 });
```

Comment:
We don't need to skip it, because there's no table context, it should work.

Reply:
Create an issue, please, and we can actually skip it. It's not a very important statement after all